src/components/Block.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import CardCanvas from "./cards/Canvas";
+import Header from "./cards/Header";
+import { Entries } from "@/types/interfaces";
+
+interface Props {
+  title: string;
+  description: string;
+  entries: Entries;
+  colSpan: number;
+  rowSpan: number;
+  blockHeight: number;
+}
+
```

Comment:
Can you put this interface in the types/interfaces file and name like BlockProps then import it here

Reply:
we thought it will be easier to view what is being passed in props without changing the file